style.css

```diff
@@ -0,0 +1,29 @@
+#title{
+    text-align: center;
+ }
+ 
+ 
+ #image{
+    max-width: 100% ;
+    height: auto;
+    display: block;
+ 
+
+ }
+ 
+ #img-div{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 50%;
+ 
+ }
+ #tribute-info{
+    text-align: center;
+ }
+ #tribute-link{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+ }
```

Comment:
I notice that your indentations and spaces are not the same. Having a code standard will help you have an organized, clean, and more readable code.

[Here is a link if you want to learn more about code standards](https://blog.codacy.com/coding-standards-what-are-they-and-why-do-you-need-them/#:~:text=What%20are%20coding%20standards%3F,to%20build%20highly%20functional%20code.)